test/unit/query-generator/describe-table-query.test.ts

```diff
@@ -57,43 +56,37 @@ describe('QueryGenerator#describeTableQuery', () => {
         + 'LEFT JOIN sys.extended_properties prop ON prop.major_id = sc.object_id '
         + 'AND prop.minor_id = sc.column_id '
         + `AND prop.name = 'MS_Description' `
-        + `WHERE t.TABLE_NAME = 'myTable'`,
+        + `WHERE t.TABLE_NAME = N'myTable'`,
       sqlite: 'PRAGMA TABLE_INFO(`myTable`);',
       db2: 'SELECT NAME AS "Name", TBNAME AS "Table", TBCREATOR AS "Schema", '
         + 'TRIM(COLTYPE) AS "Type", LENGTH AS "Length", SCALE AS "Scale", '
         + 'NULLS AS "IsNull", DEFAULT AS "Default", COLNO AS "Colno", '
         + 'IDENTITY AS "IsIdentity", KEYSEQ AS "KeySeq", REMARKS AS "Comment" '
         + 'FROM SYSIBM.SYSCOLUMNS '
         + `WHERE TBNAME = 'myTable' AND TBCREATOR = USER;`,
-      ibmi: `SELECT
-        QSYS2.SYSCOLUMNS.*,
-        QSYS2.SYSCST.CONSTRAINT_NAME,
-        QSYS2.SYSCST.CONSTRAINT_TYPE
-        FROM
-        QSYS2.SYSCOLUMNS
-        LEFT OUTER JOIN
-          QSYS2.SYSCSTCOL
-        ON
-          QSYS2.SYSCOLUMNS.TABLE_SCHEMA = QSYS2.SYSCSTCOL.TABLE_SCHEMA
-          AND
-          QSYS2.SYSCOLUMNS.TABLE_NAME = QSYS2.SYSCSTCOL.TABLE_NAME
-          AND
-          QSYS2.SYSCOLUMNS.COLUMN_NAME = QSYS2.SYSCSTCOL.COLUMN_NAME
-        LEFT JOIN
-          QSYS2.SYSCST
-        ON
-          QSYS2.SYSCSTCOL.CONSTRAINT_NAME = QSYS2.SYSCST.CONSTRAINT_NAME
-        WHERE QSYS2.SYSCOLUMNS.TABLE_SCHEMA = CURRENT SCHEMA AND QSYS2.SYSCOLUMNS.TABLE_NAME = 'myTable'`,
+      ibmi: 'SELECT '
```

Comment:
I think using a template literal makes this more readable. IIRC expectsql doesn't care too much about whitespace (I think. I remember making expectsql minify both SQLs before comparing)

Reply:
It does, should we make a template literal the default behaviour for these tests?

Reply:
I think so, it's a bit more readable

Hopefully our "minifySql" function works well